geonode_mapstore_client/client/js/components/Permissions/Permissions.jsx

```diff
@@ -188,6 +188,7 @@ function Permissions({
                                 <PermissionsRow
                                     key={group.id}
                                     {...group}
+                                    superUser={group.is_superuser}
```

Comment:
I'm not sure yet if a group can be super user because the client does not map the group with the is_superuser property.
We can remove this for now